src/store/reducers/tenants/utils.ts

```diff
@@ -22,36 +22,28 @@ const getTenantBackend = (tenant: TTenant) => {
 };
 
 export const calculateTenantMetrics = (tenant?: TTenant) => {
-    const {
-        CoresUsed,
-        MemoryUsed,
-        StorageAllocatedSize,
-        CoresLimit,
-        MemoryLimit,
-        StorageLimit,
-        Metrics = {},
-    } = tenant || {};
+    const {CoresUsed, MemoryUsed, StorageAllocatedSize, MemoryLimit, StorageLimit, PoolStats} =
+        tenant || {};
 
-    const cpuFromCores = isNumeric(CoresUsed) ? Number(CoresUsed) * 1_000_000 : undefined;
-    const cpuFromMetrics = isNumeric(Metrics.CPU) ? Number(Metrics.CPU) : undefined;
+    const systemPoolUsage = PoolStats?.find(({Name}) => Name === 'System')?.Usage;
+    const userPoolUsage = PoolStats?.find(({Name}) => Name === 'User')?.Usage;
 
-    const cpu = cpuFromCores ?? cpuFromMetrics ?? undefined;
-
-    const rawMemory = MemoryUsed ?? Metrics.Memory;
-
-    const memory = isNumeric(rawMemory) ? Number(rawMemory) : undefined;
-
-    // Blob storage - actual database size
+    const cpu = isNumeric(CoresUsed) ? Number(CoresUsed) * 1_000_000 : undefined;
+    const memory = isNumeric(MemoryUsed) ? Number(MemoryUsed) : undefined;
     const storage = isNumeric(StorageAllocatedSize) ? Number(StorageAllocatedSize) : undefined;
-    const cpuLimit = isNumeric(CoresLimit) ? Number(CoresLimit) : undefined;
+
+    const cpuUsage =
```

Comment:
Can you please add a comment, why we currently use only `System` and `User` pools?